news/utils.py

```diff
@@ -54,6 +57,26 @@ def __init__(self, message, suggestion=None):
         self.suggestion = suggestion
 
 
+def get_email_block_list():
+    """Return a list of blocked email domains."""
+    cache_key = 'email_block_list'
+    block_list = email_block_list_cache.get(cache_key)
+    if block_list is None:
+        block_list = BlockedEmail.objects.values_list('email_domain', flat=True)
```

Comment:
Huh... yeah. Good call. I was just wrong and thought that when `flat=True` was passed it returned a list object.